examples/sveltekit/.gitignore

```diff
@@ -0,0 +1,10 @@
+.DS_Store
```

Comment:
Is there any reason why this needs a separate `.gitignore` from the one at the root of the repo?

Reply:
Not at all, will remove it